ocaml/idl/json_backend/gen_json.ml

```diff
@@ -381,6 +381,32 @@ end = struct
     else
       cmp
 
+  let rec list_dedup cmp =
+    let rec loop acc = function
+      | [] ->
+          List.rev acc
+      | [item] ->
+          loop (item :: acc) []
+      | a :: b :: rest when cmp a b = 0 ->
+          loop (a :: acc) rest
+      | a :: b :: rest ->
+          loop (a :: acc) (b :: rest)
+    in
+    loop []
+
+  let remove_outdated changes =
+    (* When several lifecycles transitions for the same entity, keep the most
+       significative one and drop the rest *)
```

Comment:
significant?

Reply:
I've used latest, which is better at expressing intention